tools/theme/package.json

```diff
@@ -210,7 +210,7 @@
     },
     "types": "./src/index.d.ts",
     "customElements": "custom-elements.json",
-    "deprecationNotice": "Color Lightest and Darkest are deprecated and will be removed in a future release",
+    "deprecationNotice": "Color Lightest and Darkest and property theme in sp-theme have been deprecated and will be removed in a future release. Use property system instead of theme",
```

Comment:
Should this address lightest/darkest and their move to light/dark separately of the text to move `theme` to `system`? Seems like it might be a bit confusing to the uninitiated.